core/include/tangram/platform.h

```diff
@@ -61,6 +61,7 @@ class Platform {
     Platform();
     virtual ~Platform();
 
+    virtual void shutdown() = 0;
```

Comment:
It looks like this isn't called anywhere, am I missing something?

Reply:
Yes I've required this during scene-refactor development. It has been used in glfwApp. I'll remove the shutdown part from this PR.

Reply:
@matteblair I recalled what shutdown was good for. It's now called first in `~Map()`. Does the comment there make sense? It can probably be handled individually for each platform but at least for glfwApp it seemed appropriate to this way.